### DISCUSSION TITLE
Copying header files and libraries to NEKRS_INSTALL_DIR

### DISCUSSION THREAD

**Discussion by rickybalin** [Nek5000/nekRS#514] [Compiling] [answered]
Hello,

I noticed that after building NekRS, a number of header files and libraries are copied to $NEKRS_INSTALL_DIR/include and $NEKRS_INSTALL_DIR/lib, respectively. I am integrating a third party package into NekRS and I was able to successfully build and run an example, but I had to manually copy the necessary libraries and header files for this package into $NEKRS_INSTALL_DIR. Where can I find the code where this copy step is done so I can automate it for this package as well?

Thanks a lot,
Riccardo

**Comment by stgeke** [comment]
Why do you want to install external header files and libraries in the nekRS installation directory? I suggest to keep them separate and use the env-var `NEKRS_UDF_INCLUDES` and `NEKRS_UDF_LDFLAGS`.

**Reply by rickybalin** [reply, marked as answer]
Thanks, I wasn't aware of those env variables. I will try using those instead.
